docs/CONTRIBUTING.md

```diff
@@ -92,6 +92,10 @@ If your app has a front-end component to it, you can override the existing `inde
 
 In case your app does not contain a front-end component you should update the `index.html` file to reflect what steps a customer should perform to make the app work, once your template has been deployed.
 
+### Adding yourself to `.owners`
+
+Each app has a `.owners` file in its root directory. This file is a list of Github usernames that will be assigned as reviewers on any PR that involves changes to that app. Add your Github username to the bottom of this file, replacing the comment `# Insert your Github username here`.
```

Comment:
None of the .owners files except from the `meta-template` includes `# Insert your Github username here`. Should the rest of them have them? Or should this comment be revised